redaxo/src/core/pages/login.php

```diff
@@ -46,7 +46,7 @@ function disableLogin() {
 
 $inputGroups = [];
 $n = [];
-$n['field'] = '<input class="form-control" type="text" value="' . rex_escape($rexUserLogin) . '" id="rex-id-login-user" name="rex_user_login" autocomplete="username" autofocus />';
+$n['field'] = '<input class="form-control" type="text" value="' . rex_escape($rexUserLogin) . '" id="rex-id-login-user" name="rex_user_login" autocomplete="username webauthn" autofocus />';
```

Comment:
Wie in der PR-Beschreibung erläutert, habe ich aktuell (nur) die autofill-Variante umgesetzt, da u.a. von Google so vorgeschlagen wird.

Dabei hatte ich bei mir allerdings das folgende Problem: Da ich als Passwort-Manager eigentlich 1Password verwende, habe ich im Safari eigentlich das automatische Ausfüllen von Passwörtern deaktiviert. Somit wurde mir dann aber auch der Passkey nicht vorgeschlagen und ich hatte gar keine Möglichkeit mich einzuloggen damit.

Daher bin ich mir noch nicht sicher, ob zusätzlich ein expliziter Passkey-Button auf der Login-Seite nicht doch sinnvoll wäre. Damit würde es unabhängig von der Einstellung im Browser funktionieren. Meinungen dazu?

(1Password wird zukünftig übrigens auch Passkeys unterstützen. Ich könnte die alternativ dann also auch dort hinterlegen.)

Reply:
@alxndr-w @jelleschutter Könnt ihr erläutern, wieso ihr dafür seid? Aufgrund meines geschilderten Szenarios, oder noch andere Gründe?

Ich bin mir selbst immer noch unsicher. Optisch fände ich es eigentlich schöner, nicht noch einen separaten Button integrieren zu müssen. Und scheint Empfehlung von Google zu sein, es nur über die autofill-Methode zu implementieren (https://web.dev/passkey-form-autofill/).

Ich weiß nicht, wie üblich mein geschildertes Szenario ist. 

Angenommen wir integrieren einen separaten Button, wie würdet ihr das gestalten?
Den bisherigen Button "Login" so lassen, und einfach drunter noch einen Button "Login mit Passkey"?

Reply:
Ich finde das schlüssig argumentiert und natürlich kann es sein, dass sich andere Muster etablieren, aber so finde ich das an sich verständlich.

Wird nach Auswahl des Passkeys automatisch abgesendet? Sowas machen heutzutage viele Login-Seiten mit dem 2. Faktor.

Generell: Passkey ersetzt User/Passwort, richtig? Es ist nicht als 2. Faktor gedacht?

Reply:
> Ich finde das schlüssig argumentiert und natürlich kann es sein, dass sich andere Muster etablieren, aber so finde ich das an sich verständlich.

Du beziehst dich jetzt auf den Google-Artikel, also würdest es ohne den Extra-Button machen?
Dazu geht jedenfalls aktuell auch meine Tendenz. Und würde dann mal schauen, wie es sich bewährt, oder ob doch ein separater Button zusätzlich noch notwendig ist.

> Wird nach Auswahl des Passkeys automatisch abgesendet? Sowas machen heutzutage viele Login-Seiten mit dem 2. Faktor.

Ja so ist. Am Benutzernamensfeld wird der Passkey vorgeschlagen. Man wählt ihn aus, bestätigt ihn im Browser-Modal (TouchID o.ä.), und dann wird der Login automatisch abgesendet.

> Generell: Passkey ersetzt User/Passwort, richtig? Es ist nicht als 2. Faktor gedacht?

Ja genau, ersetzt User/Passwort.

Reply:
> Da ich als Passwort-Manager eigentlich 1Password verwende, habe ich im Safari eigentlich das automatische Ausfüllen von Passwörtern deaktiviert. 

1P klappt auch ohne Deaktivierung und nutze es auch so. Schein dann eher eine persönliche Preferenz zu sein und würde von einem zusätzlichen Button dann eher absehen.